src/com/lftechnology/java/training/srijan/basics/coreapp/dao/EmployeeDaoImpl.java

```diff
@@ -0,0 +1,433 @@
+
+package com.lftechnology.java.training.srijan.basics.coreapp.dao;
+
+import com.lftechnology.java.training.srijan.basics.coreapp.domain.Employee;
+import com.lftechnology.java.training.srijan.basics.coreapp.domain.SearchEmployee;
+import com.lftechnology.java.training.srijan.basics.coreapp.domain.UserRole;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class EmployeeDaoImpl implements EmployeeDao {
+
+	private static final Logger LOGGER =
+		Logger.getLogger(EmployeeDaoImpl.class.getName());
+	
+	@Override
+	public Boolean checkUsernameValidation(Employee e){
+		ResultSet resultSet = null;
+		Boolean userCheck = false;
+		Connection con = null;
+		con = DbConnection.dbConnection();
+		PreparedStatement prepareStatementCheckSameEmployee = null;
+		try {
+			if (con != null) {
+				String query =
+					"Select * from employeeDetails where username = ?";
+				prepareStatementCheckSameEmployee = con.prepareStatement(query);
+				prepareStatementCheckSameEmployee.setString(1, e.getUserName());
+				resultSet = prepareStatementCheckSameEmployee.executeQuery();
+				while (resultSet.next()) {
+					if (e.getUserName().equals(
+						resultSet.getString("username"))) {
+						userCheck = true;
+					}
+					else {
+						userCheck = false;
+					}
+				}
+			}
+		}
+		catch (Exception ex) {
+			LOGGER.log(Level.INFO, "Exception{0}", ex);
+		}
+		finally {
+			try {
+				resultSet.close();
+				prepareStatementCheckSameEmployee.close();
+				con.close();
+			}
+			catch (SQLException e1) {
+				LOGGER.log(Level.INFO, "SQL Exception{0}", e1);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "SQL Exception{0}" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)